src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -699,7 +699,7 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                             ->defaultValue(array('en'))
                         ->end()
                         ->booleanNode('logging')->defaultValue(false)->end()
-                        ->scalarNode('formatter')->defaultValue(class_exists(\MessageFormatter::class) ? 'translator.formatter.default' : 'translator.formatter.symfony')->end()
+                        ->scalarNode('formatter')->defaultValue('translator.formatter.default')->end()
```

Comment:
do we even need to have this config here (which is also new for now) ? I would remove it as there is only 1 meaningful value in core (and people wanting to do very custom stuff could override the service instead, as done in other parts of the framework). We don't need this config to migrate to the new API now.

Reply:
I'd love to remove it but it's there since 3.4 (we can deprecate in 4.3 though)